Directwerk/directwerk-app/src/main/resources/db/migration/V55__add_articles_feature_module.sql

```
@@ -0,0 +1,19 @@
+-- ARTICLE_RSS/ARTICLE_FEED_BUILDER previously depended directly on DIGITAL_CONTENT, so there was no
+-- way to disable the Write desk for a tenant without disabling DIGITAL_CONTENT itself (which also
+-- backs podcast bonus-content storage). Introduce a dedicated ARTICLES module, mirroring how PODCAST
+-- sits between DIGITAL_CONTENT and PODCAST_RSS/FEED_BUILDER.
+INSERT INTO feature_modules (module_key, name, depends_on, is_core, platform_active) VALUES
+    ('ARTICLES', 'Articles', '["DIGITAL_CONTENT"]', FALSE, TRUE);
+
+UPDATE feature_modules SET depends_on = '["ARTICLES"]' WHERE module_key = 'ARTICLE_RSS';
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)